test/bad_dtls_test.c

```diff
@@ -306,6 +306,9 @@ static int send_record(BIO *rbio, unsigned char type, uint64_t seqnr,
     /* Append HMAC to data */
     hmac = EVP_MAC_fetch(NULL, "HMAC", NULL);
     ctx = EVP_MAC_CTX_new(hmac);
+    if(ctx == NULL){
```

Comment:
please make your code format similar to line 300.

Reply:
'hmac' and 'enc' will be memory leaks if you just return here without freeing them.

Reply:
Thanks! That's my mistake, I will format it